installer/src/test/java/com/amazon/aws/partners/saasfactory/saasboost/CloudFormationTemplateParserTest.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.amazon.aws.partners.saasfactory.saasboost;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import org.junit.Test;
+
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import static org.junit.Assert.*;
+
+public class CloudFormationTemplateParserTest {
+
+    @Test
+    public void testReadTemplate() throws Exception {
```

Comment:
Currently this tests the jackson YAMLFactory. Why not test installers `getCloudFormationParameterMap()`?

Reply:
That was the original intent, but torn on what to do about the method arguments. Do you suggest that the caller of `getCloudFormationParameterMap` own resolving the `java.nio.file.Path` of the template file and passing it in as an argument? It's either that or the test class would have to set `workingDir` which does not currently expose a traditional setter.